packages/kotti-table/src/constants.js

```diff
@@ -20,10 +20,8 @@ export const DEFAULT_RENDER_CELL = function DEFAULT_RENDER_CELL(
 	{ row, rowIndex, column, columnIndex, value },
 ) {
 	return [
-		column.formatter(value, row, column, columnIndex, rowIndex) ||
-			(typeof column.default === 'function'
-				? column.default(value, row, column, columnIndex, rowIndex)
-				: column.default),
+		column.formatter(value, row, column, columnIndex, rowIndex) ??
```

Comment:
There’s one behavior in this line that we might want to re-consider actually. While I personally prefer only treating `null` as "empty", I think with the current table there’s a point to be made for potentially allowing `''` to count as empty as well, which this operator doesn’t do. So, maybe we should change this to `[null, undefined, ''].includes(formattedValue) ? column.default : formattedValue`.